docs/contributors/advanced/sponsor-an-upload.md

```diff
@@ -1,31 +1,97 @@
-(sponsor-an-upload)=
+(how-to-sponsor-an-upload)=
 # How to sponsor an upload
 
-Members of the {term}`Ubuntu Sponsors` and {term}`Ubuntu Security Sponsors`
-teams have the right to sponsor patches or new packages. If you are interested
-in {ref}`sponsoring <sponsorship>`, and have gained upload rights, you can
-apply to join these teams.
-
-See {ref}`path-to-upload-rights` for details on how to apply for upload rights.
+{term}`Ubuntu sponsors` and {term}`Ubuntu security sponsors` have the right to sponsor patches or new packages. If you are interested in {ref}`sponsoring <sponsorship>`, and have gained upload rights, apply to join these teams. See {ref}`path-to-upload-rights` for details on how to apply for upload rights.
```

Comment:
The change from "you can apply..." to "apply..." feels quite imperative, like you *must* do it (but I don't think it's required).